knex.ts

```diff
@@ -0,0 +1,5 @@
+import * as knex from 'knex'
+
+import * as knexConfig from './knexfile.js'
+// knexConfig.debug = true;
```

Comment:
no commented code please.
If you intend to use it on certain situations, make it be enabled depending on the environment, for example.

```
if (process.env.NODE_ENV !== 'production') {
  knexConfig.debug = true;
}
```